test/msd_test.py

```diff
@@ -296,6 +296,11 @@ def test_mass_storage(workspace, parent_test):
     test.set_flush_size(0x1000)
     test.run()
 
+    if target.name == "BlueNinja":
+        test_info.info("Skip the remaining MSD test.")
+        test_info.info("Writing incomplete binary to TZ10xx MCU, CoreSight becomes unresponsive.")
```

Comment:
Is there any documentation for why the TZ10xx becomes unresponsive when a blank binary is written?  Does this set lock settings or something like that?